packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.css

```diff
@@ -0,0 +1,30 @@
+.searchbox-container{
+    display: flex;
+    position: relative;
+    border-color: #e5e8e8;
+    height: 32px;
+    border-radius: 4px;
+    box-sizing: border-box;
+    background: #FFFFFF;
+}
+
+.searchbox_input {
```

Comment:
Using the BEM naming convention would be nice and more aligned with SLDS. Here it would mean renaming the classes like this:

`searchbox-container` --> `searchbox__container`
`searchbox_input` --> `searchbox__input`
`search-button` --> `searchbox__search-button` or even `searchbox__submit-button` to match the exposed attribute name.